release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -0,0 +1,861 @@
+
+[[ocp-311-podman]]
+==== Podman (Technology Preview)
+
+Podman is currently in xref:ocp-311-technology-preview[Technology Preview] and
+is not for production workloads.
+
+Podman is a daemon-less CLI/API for running, managing, and debugging OCI containers and pods. It:
+
+* Is fast and lightweight.
+* Leverages runC.
+* Provides a syntax for working with containers.
+* Has remote management API via Varlink.
+* Provides systemd integration and advanced namespace isolation.
+
+For more information, see link:https://blog.openshift.com/crictl-vs-podman/[Crictl Vs Podman].
+
+[[ocp-311-node-problem-detector]]
+==== Node Problem Detector (Technology Preview)
+
+The Node Problem Detector is currently in xref:ocp-311-technology-preview[Technology
+Preview] and is not for production workloads.
+
+The Node Problem Detector monitors the health of your nodes by finding certain
+problems and reporting these problems to the API server, where external
+controllers could take action. The Node Problem Detector is a daemon that runs
+on each node as a daemonSet.  The daemon tries to make the cluster aware of node
+level faults that should make the node not schedulable. When you start the Node
+Problem Detector, you tell it a port over which it should broadcast the issues
+it finds. The detector allows you to load sub-daemons to do the data collection.
+There are three as of today.  Issues found by the problem daemon can be
+classified as `NodeCondition`.
+
+Problem daemons:
+
+* Kernel Monitor: Monitors kernel log via journald and reports problems according
+to regex patterns.
+* AbrtAdaptor: Monitors the node for kernel problems and application crashes from
+journald.
+* CustomerPluginMonitor: Allows you to test for any condition and exit on a `0` or
+`1` should your condition not be met.
+
+See
+xref:../admin_guide/node_problem_detector.adoc#admin-guide-node-problem-detector[Node
+Problem Detector] for more information.
+
+[[ocp-311-cluster-autoscaling]]
+==== Cluster Autoscaling (AWS Only)
+
+You can configure an auto-scaler on your {product-title} cluster in
+Amazon Web Services (AWS) to provide elasticity for yor application
+workload. The auto-scaler ensures that enough nodes are active to run
+your pods and that the number of active nodes is proportional to
+current demand.
+
+See [Configuring the cluster auto-scaler in AWS] for more information.
+
+[[ocp-311-web-console]]
+=== Web console
+
+[[ocp-311-cluster-admin-console]]
+==== Cluster Administrator Console (Technology Preview)
+
+The Cluster Administrator Console is currently in
+xref:ocp-311-technology-preview[Technology Preview] and is not for production
+workloads.
+
+{product-title} 3.11 introduces a cluster administrator console tailored toward
+application development and cluster administrator personas.
+
+See
+xref:../install/configuring_inventory_file.adoc#install-config-configuring-inventory-file[Configuring
+Your Inventory File] for details on configuring the admin console.
```

Comment:
Didn't see ansible variables for admin console listed in [Configuring Your Inventory File](http://file.rdu.redhat.com/~ahardin/09102018/311-release-notes/install/configuring_inventory_file.html)